src/elevation/addElevation.ts

```diff
@@ -0,0 +1,21 @@
+import {coordEach} from '@turf/meta';
+import {Feature, FeatureCollection, Geometry} from 'geojson';
+import {getElevation} from './getElevation.js';
```

Comment:
for my education, why from `.js` file vs the exported function in `getElevation.ts`?

Reply:
The ECMAScript Modules specification says that imports must be fully-qualified paths. The contents of `src/` are in TypeScript, but the output needs to be ESM-compliant Javascript. 

The real reason is [obstinacy](https://stackoverflow.com/questions/75807785/why-do-i-need-to-include-js-extension-in-typescript-import-for-custom-module).